Frameworks/UIKit/UIButton.mm

```diff
@@ -187,7 +189,7 @@ - (instancetype)initWithCoder:(NSCoder*)coder {
 
 - (void)_initUIButton {
     // Store a strongly-typed backing button
-    _xamlButton = rt_dynamic_cast<WXCButton>([self xamlElement]);
+    _xamlButton = [self _xamlElementInternal].try_as<Controls::Button>();
```

Comment:
Yes, as() throws on failure.

---
In reply to: [109241845](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109241845) [](ancestors = 109241845)